zap/src/main/java/org/zaproxy/zap/extension/brk/BreakpointsOptionsPanel.java

```diff
@@ -50,7 +51,11 @@
     private JCheckBox checkBoxConfirmDropMessage = null;
     private JCheckBox checkBoxAlwaysOnTop = null;
     private JCheckBox checkBoxInScopeOnly = null;
+    private JCheckBox checkBoxShowIgnoreFilesButtons = null;
     private JComboBox<String> buttonMode = null;
+    private ZapTextField javascriptUrlRegexField = null;
+    private ZapTextField cssAndFontsUrlRegexField = null;
+    private ZapTextField multimediaUrlRegexField = null;
```

Comment:
Add before `buttonMode`?

Reply:
I can change it. Why do you suggest that?

Reply:
Keep them in order they are shown/used.